### DISCUSSION TITLE
Still see test ads and no is_real check button

### DISCUSSION THREAD

**Discussion by JUNZ1** [Poing-Studios/Godot-AdMob-Android-iOS#106] [Q&A]
Hello all, first of all in would want to thank you for this plugin, my problem is yesterday i released my app for the real banner with following steps

1 - Updated Manifest with my app id
2- Updated banner id in the godot engine plugin(there was no is_real check box)

Builded the app but still see the test banner. İs there something wrong or missing step i didnt do?

Thank you for kindly suport.

**Comment by gumaciel** [comment]
Hello, `is_real` was removed, read here: https://github.com/Poing-Studios/godot-admob-android/discussions/77#discussioncomment-2143195

Your game is already on Google Play?

**Reply by JUNZ1** [reply]
Thank you for quick reply, I just re installed now and look working currently

I am thankfull for your return
